### DISCUSSION TITLE
Experiencing intermittent 500 internal server error

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#5507] [Help] [answered]
### Summary

On my frontend-only site, users are experiencing 500 errors intermittently. [Vercel Status](https://www.vercel-status.com/) says they're all online... what can I do about this?

### Steps to Reproduce

```
500: INTERNAL_SERVER_ERROR
Code: INTERNAL_DEPLOYMENT_FETCH_FAILED 
ID: redacted
```

**Comment by swarnava** [comment, marked as answer]
Hi,

The issue should be fixed now: https://www.vercel-status.com/incidents/3ly94g3wbqtl
